pom.xml

```diff
@@ -74,6 +77,62 @@
       <version>${sonar.version}</version>
```

Comment:
I reverted the changes in pom.xml to move the sonar-plugin-api to version 5.2-RC2. I needs other to be released parent pom. Will resolve this comment once 24 version of "parent" is released today.